Students/Semester2/lessons/students/1_basics/5_lists_and_dicts/5_lists_and_dicts.ipynb

```diff
@@ -26,14 +39,15 @@
     "# and the next item is 1, followed by 2, etc.\n",
     "\n",
     "# In addition, Python supports negative indexes, in which case it counts from the end.\n",
-    "# So the last character can be indexed with -1, the second to last with -2, etc.:\n",
+    "# So the last character can be indexed with -1, the second to last with -2, etc.\n",
     "class_list = [\"Andy\", \"Bronny\", \"Carlos\", \"Denisa\", \"Enrique\"]\n",
     "#               -5       -4        -3        -2        -1\n",
     "\n",
-    "# You can access single elements from a list using the name followed by a number in [], like so:\n",
+    "# You can access single elements from a list using the name followed by a number in [].\n",
+    "#As an example:\n",
```

Comment:
there should be a space here after the pound sign

Reply:
changes have been made